cloudbuild.yaml

```diff
@@ -1,11 +1,12 @@
 # See https://cloud.google.com/cloud-build/docs/build-config
 
 # this must be specified in seconds. If omitted, defaults to 600s (10 mins)
-timeout: 1200s
+timeout: 9000s
```

Comment:
give more time. it does not (should not!) take this much time, we can watch a few runs and bump this down

Reply:
Please do remember to lower this value. :)